### PR TITLE
[helm/charts] Add konfigurator

### PR DESCRIPTION
**Pull request by faizanahmad055**
Signed-off-by: Faizan Ahmad
username: faizanahmad055
email: faizan.ahmad55@outlook.com

#### What this PR does / why we need it:
- [Konfigurator](https://github.com/stakater/konfigurator) an opensource tool maintained by [Stakater](https://github.com/stakater)
- Konfigurator is a kubernetes operator that dynamically generates and manages app configuration based on kubernetes resources

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by faizanahmad055** [comment]
Hi @mattfarina and @viglesiasce 
Ping

**Comment by faizanahmad055** [comment]
Hi @mattfarina and @viglesiasce
Ping

**Comment by faizanahmad055** [comment]
Hi @mattfarina and @viglesiasce
Can you guys please take a look, this is a very useful chart

**Comment by faizanahmad055** [comment]
Hello @mattfarina and @viglesiasce
Any update on this please

File: stable/konfigurator/values.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+konfigurator:
```
Comment (viglesiasce) [inline stable/konfigurator/values.yaml]: If this is the only component, move the values to the root level

File: stable/konfigurator/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Generated from deployments/kubernetes/templates/chart/Chart.yaml.tmpl
+
+apiVersion: v1
+name: konfigurator
+description: Konfigurator chart that runs on kubernetes
```
Comment (viglesiasce) [inline stable/konfigurator/Chart.yaml]:
```suggestion
description: Konfigurator chart that runs on Kubernetes
```

File: stable/konfigurator/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# ![](assets/web/Konfigurator-round-100px.png) Konfigurator
+
+## Problem
+
+Dynamically generating app configuration when kubernetes resources change.
+
+## Solution
+
+A kubernetes operator that can dynamically generate app configuration when kubernetes resources change
```
Comment (viglesiasce) [inline stable/konfigurator/README.md]:
```suggestion
A Kubernetes operator that can dynamically generate app configuration when Kubernetes resources change
```

File: stable/konfigurator/README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+# ![](assets/web/Konfigurator-round-100px.png) Konfigurator
+
+## Problem
+
+Dynamically generating app configuration when kubernetes resources change.
```
Comment (viglesiasce) [inline stable/konfigurator/README.md]:
```suggestion
Dynamically generating app configuration when Kubernetes resources change.
```

File: stable/konfigurator/README.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+## Solution
+
+A kubernetes operator that can dynamically generate app configuration when kubernetes resources change
+
+## Features
+
+- Render Configurations to
+  - ConfigMap
+  - Secret
+- Support for GO Templating Engine
```
Comment (viglesiasce) [inline stable/konfigurator/README.md]:
```suggestion
- Support for Go Templating Engine
```

File: stable/konfigurator/templates/role.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.konfigurator.rbac.enabled }}
+---
```
Comment (viglesiasce) [inline stable/konfigurator/templates/role.yaml]: These aren't needed

**Review by viglesiasce — CHANGES_REQUESTED** [review]

File: stable/konfigurator/values.yaml (hunk 1 above)
Comment (faizanahmad055) [reply stable/konfigurator/values.yaml]: Updated

File: stable/konfigurator/Chart.yaml (hunk 2 above)
Comment (faizanahmad055) [reply stable/konfigurator/Chart.yaml]: Updated

File: stable/konfigurator/README.md (hunk 3 above)
Comment (faizanahmad055) [reply stable/konfigurator/README.md]: Updated

File: stable/konfigurator/README.md (hunk 4 above)
Comment (faizanahmad055) [reply stable/konfigurator/README.md]: Updated

File: stable/konfigurator/README.md (hunk 5 above)
Comment (faizanahmad055) [reply stable/konfigurator/README.md]: Updated

File: stable/konfigurator/templates/role.yaml (hunk 6 above)
Comment (faizanahmad055) [reply stable/konfigurator/templates/role.yaml]: Updated

**Comment by faizanahmad055** [comment]
Hi @viglesiasce

I've implemented the changes you asked. Can you please take a look.

**Pull request closed without merging**